policy/centos7/rke2-selinux.spec

```diff
@@ -16,32 +15,42 @@ restorecon -R /var/lib/rancher; \
 restorecon -R /var/run/k3s; \
 restorecon -R /var/run/flannel
 
-
 %define selinux_policyver 3.13.1-252
 %define container_policyver 2.107-3
-
-Name:   rke2-selinux
-Version:	%{rke2_selinux_version}
-Release:	%{rke2_selinux_release}.el7
-Summary:	SELinux policy module for rke2
-
-Group:	System Environment/Base		
-License:	ASL 2.0
-URL:		http://rancher.com
-Source0:	rke2.pp
-Source1:	rke2.if
-
+%define container_policy_epoch 2
+%define container_policy_schism 2.164.2
```

Comment:
accurate name. I do wonder what happens in the long term due to this, i.e. if we will run into conflicts left-and-right trying to install due to it.

Reply:
:man_shrugging: